data/pathfinder/paizo/roleplaying_game/core_rulebook/cr__datacontrol.lst

```diff
@@ -607,6 +580,25 @@ FACTDEF:ABILITY|DwarfDefensiveTraining			DATAFORMAT:BOOLEAN	VISIBLE:YES	SELECTAB
 FACTDEF:ABILITY|DwarfStonecunning				DATAFORMAT:BOOLEAN	VISIBLE:YES	SELECTABLE:NO	REQUIRED:NO	DISPLAYNAME:Stonecunning										EXPLANATION:Remove Stonecunning					
 FACTDEF:ABILITY|DwarfWeaponFamiliarity			DATAFORMAT:BOOLEAN	VISIBLE:YES	SELECTABLE:NO	REQUIRED:NO	DISPLAYNAME:WeaponFamiliarity								EXPLANATION:Remove WeaponFamiliarity			
 
+###Block: Saves File
+#FACTDEF:CHECKS|MainStat	EXPLANATION:Output the Stat a save uses to the OS
```

Comment:
The valid target is probably "CHECK" not "CHECKS" - have you tested this data before submitting a PR?